tests/app/views/contexts/test_thank_you_context.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime, timezone
+
+import pytest
+from flask import Flask
+
+from app.data_models.session_data import SessionData
+from app.views.contexts.thank_you_context import build_thank_you_context
+
+SURVEY_TYPE_DEFAULT = "default"
+SURVEY_TYPE_SOCIAL = "social"
+SUBMITTED_AT = datetime.now(timezone.utc)
+
+
+@pytest.fixture
+def fake_session_data():
+    return SessionData(
+        tx_id="tx_id",
+        schema_name="some_schema_name",
+        language_code=None,
+        launch_language_code=None,
+        survey_url=None,
+        ru_ref="ru_ref",
+        response_id="response_id",
+        case_id="case_id",
+        period_str=None,
+        ru_name=None,
+    )
+
+
+def test_social_survey_context(fake_session_data, app: Flask):
+    with app.app_context():
+
+        context = build_thank_you_context(
+            fake_session_data, SUBMITTED_AT, SURVEY_TYPE_SOCIAL
+        )
+
+        assert context["submission_text"] == "Your answers have been submitted."
+        assert len(context["metadata"]) == 4
```

Comment:
I think this should check the number of metadata items as it did previously.

Reply:
Thanks, checking number of metadata items now!